website/static/install

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+
+set -eu
+
+INSTALLPATH=/usr/local/bin
+
+happyexit() {
+  echo "Installed version:"
+  echo "$(talosctl version 2>/dev/null )"
```

Comment:
same here, use the full path:

```suggestion
  echo "$(${INSTALLPATH}/talosctl version 2>/dev/null )"
```

Reply:
that's a good point, for me when I tested the modifed version of the script with `/tmp` it actually picked up my normal talosctl install, if the user specifies a custom path I guess it's upto them to make sure it's in their search path I guess, or maybe we could just check if there's already a talosctl in normal path and warn them of a custom path

Reply:
I think that if people are setting INSTALLPATH, they know about search paths, and would want to see the version that is called first... ANd can adjust path if necessary.
So I'd rather say , as you said " if the user specifies a custom path I guess it's upto them to make sure it's in their search path "